<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_center">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="plainTextEdit_log">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_bottom">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>72</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>72</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_get_all_stocks">
          <property name="text">
           <string>获取所有证券数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_query_sh_main">
          <property name="text">
           <string>查询沪市主板股票</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_query_sz_main">
          <property name="text">
           <string>查询深市主板股票</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_query_gem">
          <property name="text">
           <string>查询创业板股票</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_query_star">
          <property name="text">
           <string>查询科创板股票</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btn_get_daily_data">
          <property name="text">
           <string>获取日线数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_update_daily_data">
          <property name="text">
           <string>更新日线数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_query_daily_data">
          <property name="text">
           <string>查询日线数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_get_weekly_data">
          <property name="text">
           <string>获取周线数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_update_weekly_data">
          <property name="text">
           <string>更新周线数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_query_weekly_data">
          <property name="text">
           <string>查询周线数据</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
